### DISCUSSION TITLE
"Spellbook" idea

### DISCUSSION THREAD

**Discussion by Vlek** [wolandark/bash-dungeon#12] [Ideas]
After the player learns about sourcing, it might be a good idea to start using a 'spellbook' (i.e., a bash file) in order to continue the game if the player stops and comes back to it later.

Having a sourceable bash file in the main dir with aliases and exported variables would be helpful.

That way you can reset things like hp, i, PoH, PoM, etc without having to go through all of the dungeon again to re-learn all of the variables that are needed.

**Comment by wolandark** [comment]
Thats a good idea. I will create a script at a certain point in the game with a symlink to it at the Enter directory for ease of use.
